pico_project/gui.py

```diff
@@ -0,0 +1,623 @@
+#
+# Copyright (c) 2021 Raspberry Pi (Trading) Ltd.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+"""
+gui -- setup for the pico-project-generator graphical user interface
+"""
+
+from picogenlib import PicoProjectFactory, LibInfo, IDE
+import tkinter as tk
+from tkinter import messagebox as mb
+from tkinter import filedialog as fd
+from tkinter import simpledialog as sd
+from tkinter import ttk
+import sys
+import shlex
+import subprocess
+import threading
+from pathlib import Path
+
+CONFIG_UNSET = "Not set"
+
+def GetBackground():
+    return 'white'
+
+
+def GetButtonBackground():
+    return 'white'
+
+
+def GetTextColour():
+    return 'black'
+
+
+def GetButtonTextColour():
+    return '#c51a4a'
```

Comment:
These could probably be constants, rather than separate functions? :shrug:

Reply:
Cannot for the life of me remember why I did it that way.